src/common/rgbd_dataset/rgbd_dataset.py

```diff
@@ -0,0 +1,137 @@
+import pickle
+import multiprocessing
+import json
+import logging
+
+import numpy as np
+import cv2 as cv
+from glob2 import glob
+from pathlib import Path
+from glob import glob1
+
+TARGET_HEIGHT = 180
+TARGET_WIDTH = 240
+TARGET_PATH = '/mnt/huawei_dataset/anon-rgbd-dataset'
+SOURCE_PATH = '/mnt/huawei_dataset/huawei_data'
+
+
+def load_depth(filename):
+    with zipfile.ZipFile(filename) as z:
+        with z.open('data') as f:
+            line = str(f.readline())[2:-3]
+            header = line.split("_")
+            res = header[0].split("x")
+            # print(res)
+            width = int(res[0])
+            height = int(res[1])
+            depthScale = float(header[1])
+            max_confidence = float(header[2])
+            data = f.read()
+            f.close()
+        z.close()
+    return data, width, height, depthScale, max_confidence
+
+
+def prepare_depthmap(data, width, height, depthScale):
+    # prepare array for output
+    output = np.zeros((width, height, 1))
+    for cx in range(width):
+        for cy in range(height):
+            #             output[cx][height - cy - 1][0] = parse_confidence(cx, cy)
+            #             output[cx][height - cy - 1][1] = im_array[cy][cx][1] / 255.0 #test matching on RGB data
+            #             output[cx][height - cy - 1][2] = 1.0 - min(parse_depth(cx, cy) / 2.0, 1.0) #depth data scaled to be visible
+            # depth data scaled to be visible
+            output[cx][height - cy - 1] = parse_depth(cx, cy, data, depthScale)
+    return (
+        np.array(
+            output,
+            dtype='float32').reshape(
+            width,
+            height),
+        height,
+        width)
+
+
+def parse_depth(tx, ty, data, depthScale):
+    depth = data[(int(ty) * TARGET_WIDTH + int(tx)) * 3 + 0] << 8
+    depth += data[(int(ty) * TARGET_WIDTH + int(tx)) * 3 + 1]
+    depth *= depthScale
+    return depth
+
+
+def check_corrspondence(depth_frame, rgb_file_list):
+    depth_name = depth_frame.split('.depth')[0]
+    frame_name = depth_name.split('depth_')
+    rgb_frame = f'rgb_{frame_name[-1]}.jpg'
+    assert rgb_frame in rgb_file_list
+    return depth_frame, rgb_frame
+
+
+def image_resize(image_path):
+    image = cv.imread(image_path)
+    image_rotated = cv.rotate(image, cv.ROTATE_90_CLOCKWISE)
+    image_rotated = cv.resize(image_rotated, (TARGET_HEIGHT, TARGET_WIDTH))
+    return image_rotated
+
+
+def read_json(filepath):
+    with open(filepath) as json_data:
+        label_data = json.load(json_data)
+    return label_data
+
+
+source_file = f'{SOURCE_PATH}/qrcode/**/depth/'
+source_file = f'{SOURCE_PATH}'
+dataset_list = []
+proc = multiprocessing.Pool()
+for elem in Path(source_file).rglob('*/depth'):
+    proc.apply_async(dataset_list.append(elem))
+proc.close()
+proc.join()  # Wait for all child processes to close
+
+
+def process_depthmap(depthmaps):
+    split_path = depthmaps.split('/depth')[0]
+    json_path = f'{split_path}/targets.json'
```

Comment:
[nit] I think a simple distinction can make code more readable:
- suffix `_fpath` for file paths
- suffix `_dir` for directory paths
- `f` for a file object/handle

```suggestion
    json_fpath = f'{split_path}/targets.json'
```

Consider applying this suggestion to other variables